app/controllers/posts_controller.rb

```diff
@@ -1,5 +1,12 @@
 class PostsController < ApplicationController
-  def index; end
+  def index
+    @user = User.find(params[:user_id])
+    @posts = Post.where(author_id: @user.id).order(created_at: :desc)
+  end
 
-  def show; end
+  def show
+    @user = User.find(params[:user_id])
+    @post = Post.find(params[:id])
+    @comments = Comment.where(post_id: @post.id).order(created_at: :desc).limit(5)
+  end
```

Comment:
- [ ] <b>REQUIRED:</b> If you want to get the comments that belongs to this specific post, you can simply get them by: 

```ruby
@comments = @post.comments
```

  that's what the relationships(`post has_many :comments`and `comment belongs_to :post`) you have created are for.